source/_integrations/invoxia.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: "Invoxia (unofficial)"
```

Comment:
```suggestion
title: "Invoxia"
```

Reply:
Remove "unofficial" from the documentation title for invoxia integration